lib/rubocop/formatter/formatter_set.rb

```diff
@@ -88,7 +89,12 @@ def builtin_formatter_class(specified_key)
           /^\[#{specified_key}\]/.match?(key) || specified_key == key.delete('[]')
         end
 
-        raise %(No formatter for "#{specified_key}") if matching_keys.empty?
+        if matching_keys.empty?
+          similar_name = NameSimilarity.find_similar_name(specified_key, BUILTIN_FORMATTER_NAMES)
+          suggestion = %( Did you mean? "#{similar_name}") if similar_name
+
+          raise Rainbow(%(Formatter "#{specified_key}" not found.#{suggestion})).red
+        end
 
         raise %(Cannot determine formatter for "#{specified_key}") if matching_keys.size > 1
```

Comment:
Maybe in this message we should clarify that multiple formatters were matched and users need to be more specific.

Reply:
It seems that multiple formatters are never specified, and if that's the case, this logic will not be called. I will investigate separately whether it can be removed as unnecessary logic.